index.html

```diff
@@ -0,0 +1,189 @@
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <h2 class="center">About me</h2>
+      <p class="center">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="cont">
+        <button class="see center-button">Get My Resume</button>
+      </div>
+    </section>
+    <section>
+      <div class="lang img-holder">
+        <div class="circle"></div>
+        <div class="topic">Languages</div>
+        <div class="row white">
+          <div class="but2"><span class="tab">JavaScript</span></div>
+          <div class="but2"><span class="tab">Ruby</span></div>
+          <div class="but2"><span class="tab">HTML</span></div>
+          <div class="but2"><span class="tab">CSS</span></div>
```

Comment:
- [ ] I would suggest you please fix your logo and also I would suggest please increase the font weight to match the design.
- [ ] Please implement it in all the cards.
### Your design
![his](https://user-images.githubusercontent.com/81727988/219549725-45444c35-ebb8-4be5-8f45-95c8b68fcd78.png)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/219549747-46ecd9d0-5d07-496c-b978-6442705d0027.png)

Reply:
completed